src/main/java/brooklyn/entity/proxy/aws/ElbControllerImpl.java

```diff
@@ -346,15 +366,16 @@ protected void createLoadBalancer(String elbName) {
         
         LOG.debug("Creating new ELB '"+elbName+"', for server-pool "+getConfig(SERVER_POOL));
 
-        AmazonElasticLoadBalancingClient client = newClient(loc);
+        AmazonElasticLoadBalancingClient client = elbClient(loc);
         try {
             CreateLoadBalancerRequest createLoadBalancerRequest = new CreateLoadBalancerRequest();
 
             createLoadBalancerRequest.setLoadBalancerName(elbName);
             createLoadBalancerRequest.setAvailabilityZones(availabilityZoneNames);
             if (Strings.isNonBlank(elbScheme)) createLoadBalancerRequest.setScheme(elbScheme);
-            if (securityGroups != null) createLoadBalancerRequest.setSecurityGroups(securityGroups);
+            addSecurityGroups(createLoadBalancerRequest);
```

Comment:
That's a bit of a historical hang-over from an earlier commit with functionality that I have removed, and squashed the commit away.  I shall get rid of this now.